backend/src/Manager/Subscription/SubscriptionHistoryManager.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Manager\Subscription;
+
+use App\AutoMapping;
+use App\Entity\SubscriptionHistoryEntity;
+use App\Repository\SubscriptionHistoryEntityRepository;
+use App\Request\Subscription\SubscriptionHistoryCreateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+use DateTime;
+
+class SubscriptionHistoryManager
+{
+    public function __construct(private AutoMapping $autoMapping, private EntityManagerInterface $entityManager, private SubscriptionHistoryEntityRepository $subscribeHistoryRepository)
+    {
+        $this->autoMapping = $autoMapping;
+        $this->entityManager = $entityManager;
+        $this->subscribeHistoryRepository = $subscribeHistoryRepository;
+    }
+
+    public function createSubscriptionHistory($subscription):SubscriptionHistoryEntity
+    { 
+        $request = new SubscriptionHistoryCreateRequest();
+       
+        $request->setStoreOwner($subscription->getStoreOwner());
+        $request->setSubscription($subscription);
+       
+        $subscriptionHistoryEntity = $this->autoMapping->map(SubscriptionHistoryCreateRequest::class, SubscriptionHistoryEntity::class, $request);
+        
+        $subscriptionHistoryEntity->setCreatedAt(new DateTime());
```

Comment:
Why not using _Timestampable_ annotation of _Gedmo_ library for automatically set createdAt?